packages/client/src/components/blocks-workspace/blocks/settings/custom/e-charts/variant/bar-chart/DataTabStyling.tsx

```diff
@@ -242,7 +242,7 @@ export const DataTabStyling = observer(
             if (!columnsSelector || columnsSelector.length === 0) {
                 return;
             }
-            let parsedValue = JSON.parse(computedValue);
+            const parsedValue = JSON.parse(computedValue);
```

Comment:
same thing here, if we are changing the values at all, it might be better to keep it as a let. What was your logic for changing this to a const